### DISCUSSION TITLE
这个项目怎么使用呀

### DISCUSSION THREAD

**Discussion by GlynisJ** [Jacksx20/weixin_robot#1] [Q&A] [answered]
您好，这个项目要怎么运行实现需要什么环境？

**Comment by Jacksx20** [comment, marked as answer]
这个项目实践主要聚焦于微信公众号的二次开发，本次实验通过弹性云服务器 ECS、镜像服务 IMS、弹性公网 IP 搭建微信公众号处理后台，使用 Python语言编写对应的微信消息处理逻辑代码，接收从微信服务端转发过来的消息，并返回处理结果给最终用户

**Reply by GlynisJ** [reply]
OK

**Reply by Jacksx20** [reply]
后面我会写一个详细的环境介绍迭代进项目中
